vim_menutrans_helper.py

```diff
@@ -0,0 +1,228 @@
+#!/user/bin/env python3
+
+import re
+import glob
+import os
+import sys
+
+menu_commands = {"me", "menu", "noreme", "noremenu", "am",
+             "amenu", "an", "anoremenu", "nme", "nmenu",
+             "nnoreme", "nnoremenu", "ome", "omenu", "onoreme",
+             "onoremenu", "vme", "vmenu", "vnoreme", "vnoremenu",
+             "xme", "xmenu", "xnoreme", "xnoremenu", "sme", "smenu",
+             "snoreme", "snoremenu", "ime", "imenu", "inoreme", "inoremenu",
+             "cme", "cmenu", "cnoreme", "cnoremenu", "tlm", "tlmenu", "tln",
+             "tlnoremenu", "tunmenu", "tu", "popup", "popu"}
+toolbar_commands = {"tmenu", "tm"}
+exe_commands = {"exe", "exec", "execute"}
+sil_commands = {"sil", "silent", "sil!", "silent!"}
+
+def unescape_double_quotes(origin):
+    """Unescape a string inside double quotes."""
+    return (origin.replace('\\"', '"').replace("\\'", "'").replace('\\t', '\t').
+            replace('\\\\', '\\'))
+
+def unescape_single_quotes(origin):
+    """Unescape a string inside single quotes."""
+    return origin.replace("''", "'")
+
+def make_untranslated_dict(untranslation_file, untranslated_dict):
+    """Get the diffence between untranslated file and translated file."""
+    
+    # Get the untranslated files' content
+    # Encode with "latin1", because we don't care about the correctness of 
+    # non-english character
+    with open(untranslation_file, encoding="latin1") as f1:
+        for line_number, line in enumerate(f1):
+            to_be_translated_words = []
+
+            line = line.strip()
+            temp_word_list = []
+            pre_end = -1
+            top_string = line.split()
+            if len(top_string) > 0 and top_string[0] in exe_commands:
+                # Extract content inside quotes from line
+                for match in re.finditer(
+                        r"'(?:''|[^'])*'|(?:\"(?:(?:\\.)|[^\"])*\")", line):
+                    mid_part = line[pre_end + 1: match.start() - 1].strip()
+
+                    # Add a digit, namely "5", if there are variables inside 
+                    # mid_part
+                    if not ((len(mid_part) == 1 and mid_part[0] == ".") or 
+                            len(mid_part) == 0 or pre_end == -1):
+                        temp_word_list.append("5")
+                    pre_end = match.end()
+                    temp_word_list.append(line[match.start() + 1: 
+                            match.end() - 1])
+
+                temp_line = ""
```

Comment:
Ditto. Please avoid using such meaningless prefix like `temp_` to name your variables.